client/my-sites/email/email-management/index.jsx

```diff
@@ -56,13 +57,26 @@ class EmailManagement extends React.Component {
 		gsuiteUsers: PropTypes.array,
 		hasGSuiteUsersLoaded: PropTypes.bool.isRequired,
 		hasSiteDomainsLoaded: PropTypes.bool.isRequired,
+		canManageSite: PropTypes.bool.isRequired,
 		selectedSiteId: PropTypes.number.isRequired,
 		selectedSiteSlug: PropTypes.string.isRequired,
 		selectedDomainName: PropTypes.string,
 	};
 
 	render() {
-		const { selectedSiteId, selectedDomainName } = this.props;
+		const { selectedSiteId, selectedDomainName, canManageSite } = this.props;
```

Comment:
Super minor but we could order props alphabetically. Having a predictable order make it easier when you looking them up 🙂.